### ISSUE TITLE
Help investigate my code

### ISSUE THREAD

**Issue by Ajibobo** [SuperSimpleDev/html-css-course-2022#37]
lesson 5 styling not showing. Below is the HTML and CSS code as well as links.

<!DOCTYPE html>
<html>
  <head><title> Text practice</title>
    <link rel="preconnect" href="https://fonts.googleapis.com">
<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap" rel="stylesheet">

    <link rel="stylesheet" href="Styles/text.css">
  </head>
  <body>
    <p class="video-title" >
    Talking Tech and AI with Google CEO Sundar Pichai!
  </p>
  
  <p class="video-stat"> 3.4M views &middot; 6 months ago </p>
  
  <p class=" video-author"> Marques Brownlee &#10003</p>
  <p class="video-description">
    Talking tech and AI on the heels of Google I/O. Also a daily driver phone reveal from Google's CEO. Shoutout to Sundar!
  </p>
  
  <p class="apple-text">
    Shop early for the best selection of holiday favourites. <span class="shop-link">Shop now ></span>
   </p>
  <p class="apple-text">
     Shop early for the<strong class="">best selection</strong>  of <u>holiday</u>  favourites. <span class="span-example">Shop now ></span>
    </p>
  </body>
</html>

p{
  font-family: roboto;
  margin-top: 0px;
  margin-bottom: 0px;

}
.video-title{
  font-family: Arial;
  font-size: 18px;
  font-weight: bold;
  font-style: normal;
  width: 280px;
  line-height: 24px;
  margin-bottom: 0px;
}
.video-stat{
  font-family:Arial;
  font-size: 14px;
  font-style: normal;
  width: 280px;
  line-height: 26px;
  margin-top: 0px;
  margin-bottom: 20px;

}
.video-author{
  font-family: Arial;
  font-size: 14px;
  color: rgb(96,96,96);
  margin-top: 0px;
  margin-bottom: 20px;
}

.video-description{
  font-family: Arial;
  font-size: 14px;
  color: rgb(96,96,96);
  width: 280px;
  line-height: 22px;
  margin-top: 0;
  margin-bottom: 100px;

}
.Apple-text{
  margin-bottom: 50px;
  font-size: 14px;
  background-color: #e34140;
  color: white;
  text-align: center;
  padding-top: 18px;
  padding-bottom:18px ;
}
.span-example{
  color: rgb(5, 5, 5);
  margin-left: 12px;
  
  
}
.span-example:hover{
  text-decoration: underline;
  cursor:pointer;

}

.shop-link{
cursor: pointer;

}    
.shop-link:hover{
text-decoration: underline;

}

**Comment by codeaj001** [comment]
href="Styles/text.css">

Change it to

href="styles/text.css

And make sure it is the right folder you selected.

**Comment by HonDiCoder** [comment]
Yup , make sure your style folder path. Or another way you can first enter text.css and used ( crtl+c ) then your can paste it in to your code. Then you will see the exact path you put css files in.
